crmsh/bootstrap.py

```diff
@@ -1527,20 +1523,47 @@ def init_qdevice():
                             args[i + 1] = f'{sudoer}@{qnetd_addr}'
                             msg += '\nOr, run "{}".'.format(' '.join(args))
             raise ValueError(msg)
+        else:
```

Comment:
Yes, this "else" means `utils.ssh_copy_id_no_raise(local_user, qnetd_user, qnetd_addr)` success and under this block, local node already has passwordless to qnetd

Reply:
With current implementation, running

```
hanode1:~ # ssh-copy-id qnetd-node
hanode1:~ # crm cluster init -N hanode2 --qnetd-hostname qnetd-node
```

results in the public key from `hanode2` not added to the `authorized_keys` on `qnetd-node`.

Reply:
```
hanode1:~ # ssh-copy-id qnetd-node
hanode1:~ # crm cluster init -N hanode2 --qnetd-hostname qnetd-node
```
This would be fine since -N will call the join process, then `init_qnetd_remote` will copy hanode2's key to qnetd

The corner issue is:
```
hanode1:~ # ssh-copy-id qnetd-node
hanode1:~ # crm cluster init -y

hanode2:~ # crm cluster join -c hanode1 -y

hanode1:~ # crm cluster init qdevice --qnetd-hostname qnetd-node -y 
```
Then hanode2 can't access qnetd directly